### PR TITLE
improved and updated rollout history information

### PR DESCRIPTION
**Pull request by kanolato**

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -69,13 +69,11 @@ In this example:
 To create this Deployment, run the following command:
 
 ```shell
-kubectl create -f  https://k8s.io/examples/controllers/nginx-deployment.yaml
+kubectl create -f  https://k8s.io/examples/controllers/nginx-deployment.yaml --record
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/controllers/deployment.md]: `--record` is an optional flag, putting it here may be misleading.

The note callout is already mentioning the flag, right?

Comment (kanolato) [reply content/en/docs/concepts/workloads/controllers/deployment.md]: you are right @tengqm it may be misleading, im going to change it

**Pull request merged**